azkaban-common/src/main/java/azkaban/executor/container/ContainerizedDispatchManager.java

```diff
@@ -79,6 +80,10 @@ public ContainerizedDispatchManager(final Props azkProps, final ExecutorLoader e
             .getInt(ContainerizedDispatchManagerProperties.CONTAINERIZED_CREATION_RATE_LIMIT, 20));
     this.containerizedImpl = containerizedImpl;
     int rampUp = azkProps.getInt(ContainerizedDispatchManagerProperties.CONTAINERIZED_RAMPUP, 100);
+    this.allowList = new HashSet<>(azkProps
```

Comment:
For cleaner code, shall we have separate class for ContainerJobTypeCriteria? All the methods related to it will also go there.

Reply:
Done